### ISSUE TITLE
Agent type and knowledge level?

### ISSUE THREAD

**Issue by edeutsch** [NCATSTranslator/reasoner-validator#90]
So we have this guidance on using fancy annotations biolink:agent_type and biolink:knowledge_level here:
https://github.com/NCATSTranslator/ReasonerAPI/blob/1.4/ImplementationGuidance/Specifications/knowledge_level_agent_type_specification.md

And our friends at MolePro are using it already as directed:
https://arax.ncats.io/beta/?r=3d5321e2-307c-4317-a596-d731d9bfbb20

But the validator is objecting:
```
Reasoner Validator version '3.7.2' validating against TRAPI schema version 'v1.4.2' and Biolink Model version '3.5.0'.


Errors:
* Knowledge Graph Edge Attribute Type Id:
=> Edge has unknown attribute_type_id
	$ infores:chembl -> infores:molepro
		# biolink:knowledge_level:
		- edge_id: 
			PUBCHEM.COMPOUND:119182--biolink:treats->MONDO:0005570
	$ global
		# biolink:agent_type:
		- edge_id: 
			UNII:T4H8FMA7IM--biolink:treats->MONDO:0015564
	$ infores:sider -> infores:molepro
		# biolink:knowledge_level:
		- edge_id: 
			PUBCHEM.COMPOUND:4913--biolink:treats->MONDO:0005570
	$ infores:ctd -> infores:molepro
		# biolink:knowledge_level:
		- edge_id: 
			PUBCHEM.COMPOUND:12035--biolink:treats->MONDO:0005570
```

Is this stuff not yet in Biolink 3.5.0 or why is the validator complaining?

**Comment by sierra-moxon** [comment]
knowledge_level and agent_type were not included in Biolink 3.5.0 because it was deemed too disruptive before September.

**Comment by edeutsch** [comment]
ah hmm, okay, well, I don't know what the best solution is here, but at least we all know that MolePro is getting slapped with validator errors for starting to use them.

**Comment by edeutsch** [comment]
I wonder if it is not too awkward to have a custom list of exceptions in the validator.

dont_complain_about_these_attribute_type_ids = [ 'biolink:knowledge_level', 'biolink:agent_type' ]

How does that seem?

**Comment by RichardBruskiewich** [comment]
(cc: @sierra-moxon). I quietly agree with you @edeutsch.

I don't feel it is fair to punish Vlado for being so proactive with the new standards. Kudos to him!

I'll effect the code change sometime today, and issue a fresh Reasoner-Validator release (probably 3.7.3). Let me know if there are any other such use cases to cover today (although I might check the issue tracker for same... just that you know the log jams more intimately than I do at the moment).

**Comment by RichardBruskiewich** [comment]
resolved by release v3.7.3
